src/app/angular/cdk-tree/cdk-tree.component.ts

```diff
@@ -75,29 +11,32 @@ interface ExampleFlatNode {
 })
 export class CdkTreeComponent {
   static selector = 'appCdkTree';
+  private readonly treeData: ExampleFlatNode[] = this.cdkTreeService.getTreeData();
   treeControl = new FlatTreeControl<ExampleFlatNode>(
     node => node.level,
     node => node.expandable
   );
 
-  dataSource = new ArrayDataSource(TREE_DATA);
+  constructor(private cdkTreeService: CdkTreeService) {}
+
+  dataSource = new ArrayDataSource(this.treeData);
```

Comment:
Nit: Properties (at least non-function ones) should go above the constructor.

Reply:
Changed